lib/web/ui/usercontext.go

```diff
@@ -115,8 +115,7 @@ func getLogins(roleSet services.RoleSet) []string {
 	denied = apiutils.Deduplicate(denied)
 	userLogins := []string{}
 	for _, login := range allowed {
-		loginMatch, _ := services.MatchLogin(denied, login)
-		if !loginMatch {
+		if isDenied := apiutils.SliceContainsStr(denied, login); !isDenied {
```

Comment:
The code looked more robust before (MatchLogin vs Contains check), why did you elect to change it?

Reply:
Because `MatchLogin` doesn't exist anymore - it's been replaced with `services.NewLoginMatcher`. The matchers have slightly different semantics, this code is really just filtering out denied logins and doesn't need to conform to the matcher interface.

Reply:
I knew I was missing something, thanks.